src/components/radio-group/bl-radio-group.ts

```diff
@@ -100,12 +100,25 @@ export default class BlRadioGroup extends FormControlMixin(LitElement) {
     event.preventDefault();
   }
 
+  getFocusedOptionIndex() {
+    return this.focusedOptionIndex;
+  }
+
   connectedCallback(): void {
     super.connectedCallback();
 
     this.tabIndex = 0;
     this.addEventListener('focus', this.handleFocus);
     this.addEventListener('keydown', this.handleKeyDown);
+    this.addEventListener('mouseover', this.handleMouseOver);
+  }
+
+  private handleMouseOver(event: MouseEvent) {
+    const mouseOverElement = event.target;
+
+    if (mouseOverElement instanceof BlRadio) {
+      this.focusedOptionIndex = this.availableOptions.indexOf(mouseOverElement);
```

Comment:
I don't think this is a good idea. Because now we set "focused option index" without actually focusing but just hovering it.